### PR TITLE
Add Sensitive term scan

### PR DESCRIPTION
**Pull request by squeedee**

### CODE REVIEW

File: content/docs/development/tutorials/runnable.md [hunk 1]
```diff
@@ -360,7 +360,7 @@ spec:
     name: md-linting-pipelinerun
   inputs:
     repository: https://github.com/kelseyhightower/nocode # <=== new repo
-    revision: master # <=== new revision
+    revision: main # <=== new revision
```
Comment (cirocosta) [inline content/docs/development/tutorials/runnable.md:363]:
```suggestion
    revision: master # <=== new revision
```

File: content/docs/v0.3.0/tutorials/runnable.md [hunk 2]
```diff
@@ -360,7 +360,7 @@ spec:
     name: md-linting-pipelinerun
   inputs:
     repository: https://github.com/kelseyhightower/nocode # <=== new repo
-    revision: master # <=== new revision
+    revision: main # <=== new revision
```
Comment (cirocosta) [inline content/docs/v0.3.0/tutorials/runnable.md:363]:
```suggestion
    revision: master # <=== new revision
```

File: content/docs/v0.4.0/tutorials/runnable.md [hunk 3]
```diff
@@ -360,7 +360,7 @@ spec:
     name: md-linting-pipelinerun
   inputs:
     repository: https://github.com/kelseyhightower/nocode # <=== new repo
-    revision: master # <=== new revision
+    revision: main # <=== new revision
```
Comment (cirocosta) [inline content/docs/v0.4.0/tutorials/runnable.md:363]:
```suggestion
    revision: master # <=== new revision
```

File: content/docs/v0.5.0/tutorials/runnable.md [hunk 4]
```diff
@@ -360,7 +360,7 @@ spec:
     name: md-linting-pipelinerun
   inputs:
     repository: https://github.com/kelseyhightower/nocode # <=== new repo
-    revision: master # <=== new revision
+    revision: main # <=== new revision
```
Comment (cirocosta) [inline content/docs/v0.5.0/tutorials/runnable.md:363]:
```suggestion
    revision: master # <=== new revision
```

File: content/docs/development/troubleshooting.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ status:
 
 ## Common status conditions
 
-Cartographer conditions follow the
+Cartographer conditions follow the``
```
Comment (idoru) [inline content/docs/development/troubleshooting.md:34]:
```suggestion
Cartographer conditions follow the
```

**Review by idoru — APPROVED** [review]
Approved, but looks like a stray double-backtick crept in?

**Pull request merged**